_integration-schemas/github/events.md

```diff
@@ -0,0 +1,161 @@
+---
+tap: "github"
+version: "1"
+key: ""
+
+name: "events"
+doc-link: "https://developer.github.com/v3/activity/events/"
+singer-schema: "https://github.com/singer-io/tap-github/blob/master/tap_github/schemas/events.json"
+description: |
+  The `{{ table.name }}` table contains information about events in your {{ integration.display_name }} repositories.
+
+replication-method: "Full Table"
+
+api-method:
+    name: "List events"
+    doc-link: "https://developer.github.com/v3/activity/events/"
+
+attributes:
+  - name: "id"
+    type: "number"
+    primary-key: true
+    description: "The event ID."
+    foreign-key-id: "event-id"
+
+  - name: "actor"
+    type: "object"
+    description: "Information about the user that triggered an event."
+    subattributes:
+      - name: "avatar_url"
+        type: "string"
+        description: ""
+      - name: "display_login"
+        type: "string"
+        description: ""
+      - name: "gravatar_id"
+        type: "string"
+        description: ""
+      - name: "id"
+        type: "number"
+        description: ""
+      - name: "login"
+        type: "string"
+        description: ""
+      - name: "url"
+        type: "string"
+        description: ""
+  - name: "created_at"
```

Comment:
I'm pretty sure this is the table's replication key, based on this: https://github.com/singer-io/tap-github/blob/master/tap_github/__init__.py#L331

Have a dev confirm, but I think this table is incremental with `created_at` as the rep key.